### PR TITLE
test_bn_reestimation_keras acceptance test failed in AimetPro-CD-Pipeline-TF-GPU

### LINKED ISSUE #1816: test_bn_reestimation_keras acceptance test failed in AimetPro-CD-Pipeline-TF-GPU

### PR DESCRIPTION
**Pull request by quic-haijunz**
fixes #1816

### CODE REVIEW

**Review by quic-bharathr — APPROVED** [review]
Hi @quic-haijunz Please Squash-Merge this PR after @quic-sundarr approves it.

**Review by quic-sundarr — APPROVED** [review]

**Pull request merged**
